client/src/components/App.jsx

```diff
@@ -66,6 +67,20 @@ class App extends React.Component {
       });
   }
 
+  updateLike() {
+    alert("I was clicked")
```

Comment:
is this alert for testing purposes?

Reply:
yep. Just need to get this branch in with everything else.